YO30 and YO46

alou-saechao opened the pull request:
- user is able to send request
- user is able to view list of request that are pending
- user is able to approve a request
- user is able to deny a request

domshyra commented on app/scripts/app.ts:
```diff
@@ -19,6 +21,10 @@ angular.module('app', ['app.controllers','ngRoute','ngStorage','ui.bootstrap', '
+    $routeProvider.when('/ApproveDeny', {
+      templateUrl:    '/views/ApproveDenyView.html',
+      controller:     'Response.Controller',
+    });
```
May need to add resolve here so that a user cannot directly route to AppoveDeny

ladinu commented:
#peer

justinshuck commented on app/scripts/components/Request/RequestController.ts:
```diff
@@ -0,0 +1,58 @@
+
+  export interface Scope {
+    sendRequest: Function;
+    carpoolID: any;
```
We might want to change the type of carpoolID here to utilize typescript

justinshuck commented:
@alou-saechao  We'll need to pull from develop again, but other than that good work!

alou-saechao commented:
- I have added the resolve feature to the ApproveDeny page.
- Changed type of carpoolID from 'any' to 'string'.
- updated my GET and POST to use ConfigService

ianaldrighetti commented:
Great work Lou! 

#peer

alou-saechao commented:
Do we need another peer for this before we merge this into develop? Not sure who does the merge.

ianaldrighetti commented:
@alou-saechao You just need two. As long as two people have peer reviewed it you can hit the "Merge pull request button."
